packages/myst-frontmatter/src/exports/validators.ts

```diff
@@ -12,14 +12,28 @@ import type { Export } from './types.js';
 import { ExportFormats } from './types.js';
 
 const EXPORT_KEY_OBJECT = {
-  required: ['format'],
-  optional: ['template', 'output', 'id', 'name', 'renderer', 'articles', 'sub_articles'],
+  required: [],
+  optional: ['format', 'template', 'output', 'id', 'name', 'renderer', 'articles', 'sub_articles'],
```

Comment:
Is this a change for now? I thought this might be something we discussed, but aren't moving on quite yet.

Reply:
I haven't moved on inferring `format` from `template` yet, but you can now infer `format` from `output` file extension.

So, `format` is still required on the export type: https://github.com/executablebooks/mystmd/pull/817/files#diff-15a4efc7ae10ddbda5633c4ef4ae20e03dcda80cc8c5662ae0bf6cfa3b2899a9R13 (no type errors means validation always adds `format`) - but it's not required on key validation since it may be filled in based on `output`. Does that make sense...?

Reply:
When we do start inferring `format` from `template`, we will need to make `format` actually optional, even on the export type, since I doubt template loading should be part of validation...